src/dlio/odom.cc

```diff
@@ -458,7 +507,8 @@ void dlio::OdomNode::publishCloud(pcl::PointCloud<PointType>::ConstPtr published
   // published deskewed cloud
   sensor_msgs::PointCloud2 deskewed_ros;
```

Comment:
This is running in a seperate thread, what happens if the `lidarPose` updates before the stamp is computed, or the `scan_header_stamp` changes before publishing?